cmd/create/cluster/cmd.go

```diff
@@ -1424,6 +1482,26 @@ func run(cmd *cobra.Command, _ []string) {
 		)
 	}
 
+	if isSTS {
+		if mode != "" {
+			operatorroles.Cmd.Run(operatorroles.Cmd, []string{
+				clusterName,
+				mode,
+				permissionsBoundary})
+			if mode == "manual" {
+				reporter.Infof("Run the following command to create the oidc provider once operator roles are "+
+					"created:\n\n \t'rosa create oidc-provider --cluster %s' \n", clusterName)
+			} else {
+				oidcprovider.Cmd.Run(oidcprovider.Cmd, []string{clusterName, mode})
+			}
```

Comment:
I still think we should remove the validation in the `create oidc-provider` command so that we can just call
```go
operatorroles.Cmd.Run(operatorroles.Cmd, []string{clusterName, mode, permissionsBoundary})
oidcprovider.Cmd.Run(oidcprovider.Cmd, []string{clusterName, mode})
```